utils/demo/helpers/createInfoSection.ts

```diff
@@ -1,14 +1,81 @@
-function createInfoSection(parent: HTMLElement) {
-  const info = document.createElement('div');
-  parent.appendChild(info);
+type InfoSectionOptions = {
```

Comment:
If you are going to do things this way, then please add default sections for the navigation options - you can see that I added a utility here: https://github.com/cornerstonejs/cornerstone3D/pull/988/files#diff-f93b74e81121da684a91900c29a11a054ad7ec48e01a8ae4ff6a569c78e0a53b
that adds default behaviour for navigation so that we can try to make it consistent.  Depending on merge order, that might end up being available, and if so, using it allows for consistent binding.